### PR TITLE
Improve documentation around std::chrono::duration<> [skip ci]

### PR DESCRIPTION
**Pull request by coryan**

### CODE REVIEW

File: bigtable/admin/column_family.h [hunk 1]
```diff
@@ -44,24 +44,27 @@ class GcRule {
...
    *
-   * The function accepts any instantiation of std::chrono::duration<> for the
+   * The function accepts any instantiation of `std::chrono::duration<>` for the
    * @p duration parameter.  For example:
    *
    * @code
    * auto rule1 = bigtable::GcRule::MaxAge(std::chrono::hours(48));
    * auto rule2 = bigtable::GcRule::MaxAge(std::chrono::seconds(48 * 3600));
-   * @encode
+   * @endcode
```
Comment (mbrukman) [inline bigtable/admin/column_family.h]: I think this typo may be causing a build failure for the docs build; can you please move this (and similar change below) to the other PR to ensure it fixes it, and keep this change just for adding no-testing-needed code formatting?

Comment (coryan) [reply bigtable/admin/column_family.h]: Done in #113.  Please merge that one first, I will rebase and resubmit.

**Comment by mbrukman** [comment]
Now that PR #113 was merged, please rebase this PR.

**Comment by coryan** [comment]
PTAL.

**Pull request merged**